.github/libcxx-setup.sh

```diff
@@ -1,27 +1,29 @@
 #!/usr/bin/env bash
 
+set -e
+
 # Checkout LLVM sources
-#git clone --depth=1 https://github.com/llvm/llvm-project.git llvm-project
-#
+git clone --depth=1 https://github.com/llvm/llvm-project.git llvm-project
```

Comment:
One more thing to note, it is rather issue-prone to use libc++ from trunk,
e.g. because nowadays they only guarantee building libc++ with clang
from the same revision or so, so building with older clang may fail,
and i'm not sure that will be caught at cmake time.
So it would be best to checkout some specific tag.